rust/src/domains/polars/csv/mod.rs

```diff
@@ -13,37 +16,37 @@ use crate::{
 #[cfg(feature = "ffi")]
 mod ffi;
 
-use super::LazyFrameDomain;
-
 pub trait DatasetMetric: Metric<Distance = IntDistance> {}
 impl DatasetMetric for SymmetricDistance {}
 impl DatasetMetric for InsertDeleteDistance {}
 impl DatasetMetric for ChangeOneDistance {}
 impl DatasetMetric for HammingDistance {}
 
-#[derive(Clone, PartialEq, Debug)]
-pub struct CsvDomain {
-    pub lazyframe_domain: LazyFrameDomain,
+#[derive(Clone)]
+pub struct CsvDomain<F: Frame> {
+    pub frame_domain: FrameDomain<F>,
     pub delimiter: char,
     pub has_header: bool,
     pub skip_rows: usize,
     pub comment_char: Option<char>,
     pub quote_char: Option<char>,
     pub eol_char: char,
     pub null_values: Option<NullValues>,
+    pub null_value_repr: String,
 }
 
-impl CsvDomain {
-    pub fn new(lazyframe_domain: LazyFrameDomain) -> Self {
+impl<F: Frame> CsvDomain<F> {
+    pub fn new(frame_domain: FrameDomain<F>) -> Self {
         CsvDomain {
-            lazyframe_domain,
+            frame_domain,
             delimiter: ',',
             has_header: true,
             skip_rows: 0,
             comment_char: None,
             quote_char: Some('"'),
             eol_char: '\n',
             null_values: None,
+            null_value_repr: "None".to_string(),
```

Comment:
The default should be the empty string:
https://pola-rs.github.io/polars/py-polars/html/reference/api/polars.DataFrame.write_csv.html